src/inference/include/openvino/runtime/core.hpp

```diff
@@ -4,7 +4,7 @@
 
 /**
  * @brief A header file for the OpenVINO Runtime Core class C++ API.
- *
+ * @ingroup ie_cpp_api
```

Comment:
we must use different groups for IE and OpenVINO APIs: 
- ie_cpp_api for IE API, such API must not be visible in documentation.
- ov_cpp_api for OpenVINO Runtime C++ API, which must be visible in documentation.